pkg/koordlet/resourceexecutor/reader.go

```diff
@@ -198,6 +199,22 @@ func (r *CgroupV1Reader) ReadCPUTasks(parentDir string) ([]int32, error) {
 	return readCgroupAndParseInt32Slice(parentDir, resource)
 }
 
+func (r *CgroupV1Reader) ReadMemoryColdPageUsage(parentDir string) (uint64, error) {
+	resource, ok := sysutil.DefaultRegistry.Get(sysutil.CgroupVersionV1, sysutil.MemoryIdlePageStatsName)
+	if !ok {
+		return 0, ErrResourceNotRegistered
+	}
+	s, err := cgroupFileRead(parentDir, resource)
+	if err != nil {
```

Comment:
directly return the `cgroupFileRead` error without wrapping
otherwise the returned error cannot be checked by `IsCgroupDirErr`
same below